### ISSUE TITLE
Comission value from paper trading parsed incorrect

### ISSUE THREAD

**Issue by BusinessDuck** [stoqey/ib#209]
Typescript typings are incorrect for `OrderState`, comissionCurrency incoming is `1.7976931348623157E308`, this an incorrect number for javascript. Also comission value is empty.

```
const openOrderHandler = (orderId: Number, contract: Contract, filled: Order, state: OrderState) => {});

this.api.on(EventName.openOrder, openOrderHandler);
```

Incorrect typing
```
    /** The order's generated commission. */
    commission?: number;
    /** The execution's minimum commission. */
    minCommission?: number;
    /** The executions maximum commission. */
    maxCommission?: number;
    /** The generated commission currency. */
    commissionCurrency?: string;
```

Looks like a parsing problem also

**Comment by BusinessDuck** [comment]
Hotfix on client side:

```javascript
commission: {
    currency: opts.currency,
    value: Number(state.commissionCurrency.split('E')[0]),
},
```

**Comment by rylorin** [comment]
Probably fixed with PR #216
